src/stdlib/pylib.ts

```diff
@@ -62,14 +74,20 @@ export function __py_multiplier(x: Value, y: Value) {
   if (typeof x === 'number' && typeof y === 'bigint') {
     return x * Number(y)
   }
-  return NaN
+  if (typeof x == 'number' && __is_string(y)) {
+    return y.repeat(x)
+  }
+  if (typeof y == 'number' && __is_string(x)) {
+    return x.repeat(y)
+  }
```

Comment:
These will never run because errors will be thrown above when `__is_numeric` gives false right?

Reply:
Oops you're right. Need to fix that.